discovery-utils/src/discovery/v0.rs

```diff
@@ -30,6 +30,19 @@ pub mod register_discovery_handler_request {
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct Empty {}
 #[derive(Clone, PartialEq, ::prost::Message)]
+pub struct QueryDeviceInfoRequest {
+    ///the basic device information that discovery handler can get is used in this request, such as device ipaddress, mac address etc. It will be a JSON string and it will be directly send to external device inventory system REST API. It is external device inventory system's resonsibility to understand this query_device_payload JSON string and respond.
```

Comment:
Where are we defining the expected JSON string per protocol? Do we expect this to change?

Reply:
query_device_payload JSON strcture is defined in each discovery handler and it will be different for different protocol. for the current supported protocols, it is defined in each discovery_handler.rs. for example in udev protocol,
https://github.com/project-akri/akri/blob/8dc09fdc891659cc1d0ff60da9630ebb5aa1d92b/discovery-handlers/udev/src/discovery_handler.rs#L26-L31